environments/prod/prod.tfvars

```diff
@@ -14,7 +14,7 @@ cdn_sku    = "Standard_Verizon"
 shutter_rg = "shutter-app-prod-rg"
 
 cft_apps_ag_ip_address = "10.13.32.122"
-cft_apps_cluster_ips   = ["10.13.15.250"]
```

Comment:
Before merging this . please make sure that the apps in the prod-00 cluster are all up and running fine.